spec/models/upload_certificate_event_spec.rb

```diff
@@ -6,8 +6,9 @@
   root = PKI.new
   entity_id = 'http://test-entity-id'
   good_cert_value = root.generate_encoded_cert(expires_in: 2.months)
+  team = Team.create(name: SecureRandom.alphanumeric, id: SecureRandom.uuid)
```

Comment:
```suggestion
  team = create(:team, name: SecureRandom.alphanumeric)
```

Reply:
I wish.

`create` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc).